build.gradle.kts

```diff
@@ -79,6 +81,30 @@ subprojects {
                 if (file.length() == 0L) throw IllegalStateException("Distribution file: $distributionFilePath is empty")
             }
         }
+
+        apply<ReleasePlugin>()
+
+        configure<PublishExtension> {
+            userOrg = Config.Sentry.userOrg
+            groupId = project.group.toString()
+            publishVersion = project.version.toString()
+            desc = Config.Sentry.description
+            website = Config.Sentry.website
+            repoName = Config.Sentry.javaRepoName
```

Comment:
Ok now I see the difference between Java and Android repo.

Reply:
What is the difference? We moved from https://github.com/getsentry/sentry-android into this one, isnt' it all the same now?

Reply:
Oh, is this a bintray repo?

Reply:
if so could it be named `bintrayRepo` please?

Reply:
no, java packages go under sentry-java, android packages go sentry-android and its not possible to change, we'd lose jcenter sync cus its a bintray limitation

Reply:
yeah I realize that on the second comment.
I suggested rename the variable. `repoName` is very ambiguous